kymatio/scattering2d/scattering2d.py

```diff
@@ -103,8 +103,8 @@ def build(self):
         # Create the filters
         self.M_padded, self.N_padded = compute_padding(self.M, self.N, self.J)
         filters = filter_bank(self.M_padded, self.N_padded, self.J, self.L)
-        self.Psi = filters['psi']
-        self.Phi = [filters['phi'][j] for j in range(self.J)]
+        self.Psi = convert_filters(filters['psi'])
+        self.Phi = convert_filters([filters['phi'][j] for j in range(self.J)])
```

Comment:
It does not because psi and phi are stored differently.. yet this is an implementation detail.. this step could be performed in filter bank! Here sounds good to me however.